activestorage/lib/active_storage/downloader.rb

```diff
@@ -35,7 +35,7 @@ def download(key, file)
       end
 
       def verify_integrity_of(file, checksum:)
-        unless ActiveStorage.checksum_implementation.file(file).base64digest == checksum
+        unless service.file(file) == checksum
```

Comment:
`file` doesn't look like a good name. Perhaps `file_checksum` or something?

Reply:
`file_base64digest` or `base64digest_file` may be more clear. It is adding a `base64digest` call onto the digest class's `file` method.